Apps/Common/SIMCoupled.h

```diff
@@ -53,7 +53,13 @@ template<class T1, class T2> class SIMCoupled
   }
 
   //! \brief Computes the solution for the current time step.
-  virtual bool solveStep(TimeStep& tp, bool firstS1 = true)
+  virtual bool solveStep(TimeStep& tp)
+  {
+    return this->solveStep(tp, true);
+  }
+
+  //! \brief Computes the solution for the current time step.
+  virtual bool solveStep(TimeStep& tp, bool firstS1)
```

Comment:
Because solveStep(tp) is ambious, it can be resolved as either a literal solveStep(tp) or solveStep(tp,true).